processor/internal/trackingplan_validation/trackingplan_validation_test.go

```diff
@@ -0,0 +1,653 @@
+package trackingplan_validation
```

Comment:
```suggestion
package trackingplan_validation_test
```

To enforce testing on exported methods